front/app/modules/commercial/content_builder/admin/components/CraftComponents/Accordion/index.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+// components
+import QuillEditedContent from 'components/UI/QuillEditedContent';
+import QuillEditor from 'components/UI/QuillEditor';
+
+// craft
+import { useNode } from '@craftjs/core';
+import {
+  Box,
+  Accordion,
+  Title,
+  IconTooltip,
+  Input,
+  Label,
+  Toggle,
+} from '@citizenlab/cl2-component-library';
+
+// hooks
+import { useTheme } from 'styled-components';
+
+// intl
+import messages from '../../../messages';
+import { FormattedMessage, injectIntl } from 'utils/cl-intl';
+
+const SimpleAccordion = ({
+  text,
```

Comment:
Had to call this something other than "Accordion" since I imported the Accordion from the component library. I'm open to using a different name though.